src/svc/invocable-scripts/schemas/Scripts.yml

```diff
@@ -1,6 +1,6 @@
 type: object
 properties:
-  runs:
+  scripts:
```

Comment:
@aanthony1243 So `runs` (or log/number/list of invocations) isn't supported right now?